docs/demo.md

```diff
@@ -0,0 +1,195 @@
+- GenAI credentials
+- mvn
+- Quarkus 3.10
+- Java 17
+
+## Step 1: Setup
+
+In this step, we will configure the Kai IDE plugin within VSCode to showcase the capabilities of Kai. For this demo, we will be using the model `meta-llama/llama-3-70b-instruct` within the Kai configuration. This setup ensures that the demo runs seamlessly within the VSCode environment, highlighting the efficiency and power of Kai for code migration in the context of our specific scenario.
+
+### Setup Kai VSCode IDE plugin
+
+- Follow along the steps listed in [here](https://github.com/konveyor-ecosystem/kai/tree/main/ide) to intergrate Kai IDE plugin with VSCode.
+
+- Before starting the Kai server, Select model `meta-llama/llama-3-70b-instruct` by adding this to `kai/config.toml` file
+
+```yaml
+provider = "IBMOpenSource"
+args = { model_id = "meta-llama/llama-3-70b-instruct", max_new_tokens = 2048 }
+```
+
+If you are interested in using `gpt-3.5-turbo` refer to the section below,
+
+#### Running Kai with GPT-3.5
+
+- Export your OpenAI api key `export OPENAI_API_KEY="<your-openai-api-key>"`
+
+- Before starting the Kai server, Select model `gpt-3.5-turbo` by adding this to `kai/config.toml` file
+
+```yaml
+provider = "OpenAI"
+args = { model_id = "gpt-3.5-turbo" }
+```
+
+- Run the server using `make run-server`
+
+## Step 2: Demo
+
+As mentioned earlier, in this demo we will show various capabilities of Kai. We will begin with the example - Change import namespaces. Kai uses hints from the anlaysis results and replaces the javax persistence with Jakarta persistence libraries in the following files:
+
+- `src/main/java/com/redhat/coolstore/model/ShoppingCart.java`
+- `src/main/java/com/redhat/coolstore/model/InventoryEntity.java`
+
+Moving on to slightly more involved tasks, Kai will be migrating the file to use the correct Quarkus cdi bean.
+
+- `src/main/java/com/redhat/coolstore/service/CatalogService.java`
+
+Lastly, for the example - EJB Remote and Message Driven Bean(MBD) changes , we will use the following two files, and we will dive deeper into it later in the demo
+
+- `src/main/java/com/redhat/coolstore/service/ShippingService.java`
+- `src/main/java/com/redhat/coolstore/service/ShoppingCartOrderProcessor.java`
+
+Through these examples, we aim to provide insights into Kai's ability to address varying degrees of migration challenges within application codebases.
+
+### 2.1 Clone the coolstore app
+
+In this step, we will clone the Coolstore application, which we will be used demo the migration process to Quarkus.
+
+1. To clone the Coolstore demo from its repository -
+
+   ```bash
+   git clone https://github.com/konveyor-ecosystem/coolstore.git
+   ```
+
+2. Next, switch to the branch of the Coolstore app that's been partially migrated -
+
+   ```bash
+   git checkout partial-migration
+   ```
+
+### 2.2 Run Analysis
+
+We will kick off by analyzing the Coolstore application using the following targets: cloud-readiness, Jakarta EE, and Quarkus.
```

Comment:
The names the IDE shows are a little different so probably helps to be explicit:
- containterization
- jakarta-ee
- jakarta-ee8+
- jakarata-ee9+
- quarkus 

I am also not 100% sure we need all 3 of the jakarta-ee targets, but I think the javax->jakarta are only in those and not in 'quarkus', but I didn't confirm.